### DISCUSSION TITLE
Migrate/upgrade from original Trilium Docker

### DISCUSSION THREAD

**Discussion by Firesphere** [TriliumNext/Notes#66] [Q&A / Support] [answered]
I'm currently not near/able to log in to my Trilium instance at home, but it's a docker setup on Zadam/Trilium.

How easy/hard is it to upgrade from Zadam/trilium to TriliumNext via Docker compose file? Is it a matter of replacing the chosen docker container, or is there more involved?

Whichever route needs to be taken, I believe it would be useful to create a pinned issue/discussion or note in the readme about this.

**Comment by meichthys** [comment]
TriliumNext is not yet ready for public use. Follow our releases to wait for a release in the coming days\weeks.

We will provide migration instructions with the releases.

**Comment by capi** [comment]
Related question: do you intend to keep a migration path from zadam/trilium for the foreseeable future, or is this a one-time thing for the first release?

**Reply by meichthys** [reply]
No promises, but yes, there will likely be a migration path at least in the short-term.

**Comment by diyoyo** [comment]
I guess this also answers this #63 , right?

**Comment by mremptypants** [comment]
any update on this

**Reply by eliandoran** [reply, marked as answer]
@mremptypants , we've just released a new stable version. The migration path is simply switching to the new DockerHub image: https://hub.docker.com/r/triliumnext/notes

It should be fully compatible and there is no difference in sync or communication protocol. You can check the release notes for more information. 

As always, manual backups are a good precaution.

**Comment by mremptypants** [comment]
OK thank you, got it worked out, had to edit my build script and not just replace the image in the docker.

**Reply by eliandoran** [reply]
@mremptypants , could you please offer more info on what changes you had to do? I'm not sure what build script you're referring to.
